### ISSUE TITLE
New Handbook Format

### ISSUE THREAD

**Issue by 42piratas** [AkashaProject/pm#44]
**Please, add your suggestions or any references to the comments below**

**Comment by 42piratas** [comment]
From Mariana:

> I listed below some references for the Handbook with different approaches:
> https://www.atlassian.com/team-playbook
> https://handbook.bakkenbaeck.com/
> https://trello.com/teams
> https://steamcdn-a.akamaihd.net/apps/valve/Valve_NewEmployeeHandbook.pdf
> 
> Regarding the new elements that we will probably need (White Paper, Token and FAQs), some of them I already considered in initial structure but I will add a Token section as well. If you have a good reference to this type of content (white paper, token) that you could share it would be really helpful.

**Comment by 42piratas** [comment]
*Important note:* whatever the approach we will use, it must be something easy and quick to update, and that allows a review process, to give everyone the means to contribute as much as they wish.

**Comment by marianagomes** [comment]
@42piratas Regarding the handbook, I talked with the team and another ideia to build this section was using Gatsby (https://www.gatsbyjs.org/) that is a static site generator but not so hard like the other example that we used for Documentation section. Of course this would be dependent on how the site would be developed, but it could be allows to present this section with a more free structure and play around with the different sections.

**Comment by 42piratas** [comment]
@marianagomes Great suggestion. And you're right, it all depends on a holistic plan, since everything must have the same visual identity, and better if we could use the same tool for docs and handbook etc, as I mentioned on https://github.com/AkashaProject/pm/issues/38 - unless we have some specific awesome features that would make sense only for one of the sections or so.

**Comment by marianagomes** [comment]
@42piratas  Regarding the Handbook we were working on two different approaches. We could follow the static site generator like GitBook and have a result similar to this: http://significa.pt/drops/2018-07-11_17-57-17_x320s.png

But since we will have quite a volume of information, maybe it could be a good idea to have the first page with the main sections, like About Akasha, highlight to AKASHA Manifesto and Contributing and three links at the bottom to other sections: How we work / Code Review / Vocabulary & Library. http://significa.pt/drops/Handbook.png

Looking to the actual structure of Handbook we can easily group some of this sub-sections in main section, so we could have a main page where the user could see the general information and possibly redirect to more specific content. Following this approach we could have two pages with sub-sections like this:
http://significa.pt/drops/2018-07-11_18-02-37_1mxfq.png
http://significa.pt/drops/2018-07-11_18-02-54_l0urj.png

The second approach is also easily updated and changeable, with possible continuous deployment and automatically generated markdown.

**Comment by 42piratas** [comment]
@marianagomes I like the idea of dividing it into sections. But keep in mind that must be in a way that we can easily expand and modify things at any time, e.g. at anytime one could add or remove a new section or sub-section, therefore we need to plan in advance to not break the UI when that happens.

**Comment by marianagomes** [comment]
@42piratas from what I understood, using this kind of tools we can easily update the content, add one more section or remove it. Both _Gatsby_ and _Hugo_, allows the same UI customization, so I think we can come up with the tool that the developers knows best, or what they think is more appropriate.

**Comment by 42piratas** [comment]
@marianagomes Yes, the tools allow us that. What I meant is desing-wise, e.g h[ere where you have a horizontal menu](https://significa.pt/drops/2018-07-11_18-02-37_1mxfq.png), we need to make sure the design won't break with extra menu items.

**Comment by marianagomes** [comment]
@42piratas  I understand your concern. Probably I’ll need help from the developers to guarantee that this approach could be easily updated, but from our side, we will develop the layout in a more modular and adaptable way and always predict how it can grow in order to maintain the design.
